kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/KieServerConstants.java

```diff
@@ -110,6 +110,8 @@
     public static final String KIE_SERVER_PARAM_RESET_BEFORE_UPDATE = "KieServerResetBeforeUpdate";
 
     public static final String KIE_SERVER_ROUTER = "org.kie.server.router";
+    public static final String KIE_ROUTER_MANAGEMENT_PASSWORD = "org.kie.router.management.password";
```

Comment:
I think it may be better to use org.kie.server.router.management.password to keep the same prefix as for other router related options.